File: Fbtc-Api/Fbtc.Domain/Entities/Endereco.cs

namespace Fbtc.Domain.Entities
{
    public class Endereco: EnderecoCep
    {
        public int EnderecoId { get; set; }
        public int PessoaId { get; set; }
        public string Numero { get; set; }
        public string Complemento { get; set; }
        public string TipoEndereco { get; set; }
        public string OrdemEndereco { get; set; }

    }
}

File: fbtc-adm/src/app/components/shared/model/endereco.ts
import { EnderecoCep } from './endereco-cep';

export class Endereco extends EnderecoCep {
    enderecoId: number;
    pessoaId: number;
    numero: string;
    complemento: string;
    tipoEndereco: string;
    ordemEndereco: string;
}
